apps/plugin-browser/.env

```diff
@@ -0,0 +1,2 @@
+API_ORIGIN=http://localhost:5001
+FRONTEND_ORIGIN=http://localhost:3000
```

Comment:
Is there a particular reason we're using `3000` as the port? Might be nice to use one that doesn't clash with the HASH frontend when developing locally

Reply:
This is the HASH frontend... it's used to generate the links for sign in / sign up if the user isn't logged in, and to the entities page etc